R/pkg/inst/worker/daemon.R

```diff
@@ -31,7 +31,15 @@ inputCon <- socketConnection(
     port = port, open = "rb", blocking = TRUE, timeout = connectionTimeout)
 
 while (TRUE) {
-  ready <- socketSelect(list(inputCon))
+  ready <- socketSelect(list(inputCon), timeout = 1)
```

Comment:
So, I intended to terminates the children for every second _or_ before launching other workers.